e2e-tests/playwright/support/ui/components/channels/post_create.ts

```diff
@@ -5,12 +5,12 @@ import {expect, Locator} from '@playwright/test';
 
 export default class ChannelsPostCreate {
     readonly container: Locator;
-
     readonly input;
 
     readonly attachmentButton;
     readonly emojiButton;
     readonly sendMessageButton;
+    readonly scheduleDraftMessageButton;
```

Comment:
I'm seeing different variants such as:
```
scheduleMessage...
scheduledMessage...
scheduleDraftMessage...
scheduledDarft...
```

I wonder if this should follow a common prefix only for consistency and easier to reuse/remember.

Reply:
Got it. I'll use scheduledDraft overall.